src/higley_lab_to_nwb/lohani_2022/utils/lohani_2022_utils.py

```diff
@@ -27,20 +31,76 @@ def get_tiff_file_paths_sorted_by_channel(folder_path: str, start_frame_index: i
     return selected_tiff_file_paths
 
 
-def create_tiff_stack(folder_path: str, output_file_path: str, start_frame_index: int = 0, frame_side: str = "left", stub_test: bool = False):
-    
+def create_tiff_stack(
+    folder_path: str,
+    output_file_path: str,
+    start_frame_index: int = 0,
+    frame_side: str = "left",
+    stub_test: bool = False,
+):
     selected_tiff_file_paths = get_tiff_file_paths_sorted_by_channel(
-        folder_path=folder_path, start_frame_index=start_frame_index, stub_test = stub_test
+        folder_path=folder_path, start_frame_index=start_frame_index, stub_test=stub_test
     )
     frames = [tifffile.imread(file_path) for file_path in selected_tiff_file_paths]
 
     if frame_side == "left":
-        stack = np.stack([frame[:, :512].transpose(1,0) for frame in frames], axis=0)
+        stack = np.stack([frame[:, :512].transpose(1, 0) for frame in frames], axis=0)
     elif frame_side == "right":
-        stack = np.stack([frame[:, 512:].transpose(1,0)  for frame in frames], axis=0)
+        stack = np.stack([frame[:, 512:].transpose(1, 0) for frame in frames], axis=0)
     else:
         raise ValueError("frame_side must be either 'right' or 'left'")
 
     tifffile.imwrite(output_file_path, stack)
 
-    return 
\ No newline at end of file
+    return
+
+
+def process_event_times(rising_times: np.ndarray, falling_times: np.ndarray):
+    # TODO define your custom function to processed event times extracted from Spike2 signal
+    return rising_times, falling_times
```

Comment:
Not sure I understood this; so the idea is that they can define they own function to get the start and stop times for `VisualStimulusInterface`?

Reply:
Yes, we talked about in the progress meeting, and we decide to go for this custom approach. For now I am using the pre-processed timestamps in the matlab file

Reply:
Got it, maybe adding this information to the README.md would be useful as well